Update local testing page to address upgrade to GameLift SDK v5 (#2415)
o3de/o3de.org: content/docs/user-guide/gems/reference/aws/aws-gamelift/local-testing.md

With GameLift Anywhere, you can verify the following:

*   Your game server is correctly integrated with the Server SDK and is properly communicating with the GameLift service to start new game sessions, accept new players, and report health and status.
    
*   Your game client is correctly integrated with the AWS SDK for GameLift and is able to retrieve information on existing game sessions, start new game sessions, join players to games and connect to the game session.
    

## 1. Create a Location

Create a location for your custom resources. A custom location labels the location of your hardware that Amazon GameLift uses to run your games in Anywhere fleets.
When creating your custom location, the location name must start with `custom-`.

```sh
aws gamelift create-location --location-name custom-location-1 --region <Region>
```

## 2. Create an Anywhere Fleet

Creating an Anywhere fleet is a much faster process compared to creating a regular AWS fleet, which usually takes about an hour to setup.

```sh

> aws gamelift create-fleet --name AnywhereFleet --compute-type ANYWHERE --locations Location=custom-location-1 --region <Region>

Reviewer: Add a note to save the Fleet-Id for the next steps (maybe provide an example of what a fleet id looks like)

If your machine is accessible via a public IP address, change that value as appropriate.

```sh
aws gamelift register-compute --compute-name CustomCompute1 --fleet-id <FleetId> --ip-address 127.0.0.1 --location custom-location-1 --region <Region>
```

Also retrieve the compute auth token as it will be necessary for the next steps.

```sh
aws gamelift get-compute-auth-token --fleet-id <FleedId> --compute-name CustomCompute1
```

## 4. Start an instance of the Game Server executable on your machine

When you are ready to test with GameLift, opt-in by setting the `sv_gameLiftEnabled` and `sv_gameliftAnywhereEnabled` CVARs to `true`. These CVARs are off by default to facilitate local testing prior to the integration of the GameLift Server SDK into your game server.

You will also need to fill in the server parameters via the following CVARs:
- `sv_gameliftAnywhereWebSocketUrl`
- `sv_gameliftAnywhereAuthToken`
- `sv_gameliftAnywhereFleetId`
- `sv_gameliftAnywhereHostId`
- `sv_gameliftAnywhereProcessId`

All of the values for these properties are retrieved in the previous steps, aside from these notes:

> - In the `HostId` property should be filled with the `ComputeName`;

Reviewer 2: ```suggestion
- The `HostId` property should be set to the `ComputeName`.
```
